rtloader/appveyor.yml

```diff
@@ -12,7 +12,7 @@ environment:
   GOPATH: C:\gopath
   # Give hints to CMake to find Pythons
   Python2_ROOT_DIR: C:\Python27-x64
-  Python3_ROOT_DIR: C:\Python37-x64
+  Python3_ROOT_DIR: C:\Python38-x64
```

Comment:
this appears to be 3.8.0 (not 3.8.1).  I'm not sure it matters, but to get 3.8.1 we'd have to install our own python at test-time.

Reply:
Yup I've been with the best-effort for appveyor. Would it be complicated to have a 3.8.1? They'll probably update it soon anyway?